### DISCUSSION TITLE
LUFS setting for EBU R 128

### DISCUSSION THREAD

**Discussion by garuffalo** [DeaDBeeF-Player/deadbeef#3002] [Q&A]
The official default LUFS target level setting for EBU R 128 is, I believe, -23 LUFS. Is that the default setting in Deadbeef for ReplayGain scanning? I ask because some other music players, e.g., MusicBee, use -18 LUFS in the calculation (also based on EBU R 128 standard).

I scanned a test album of m4a files in Deadbeef and compared the ReplayGain tags to the ones created in MusicBee, and although most tracks had very similar values, some tracks were significantly different. The only thing I could think of is that Deadbeef used a different target level.

I have a large collection of files that were previously scanned in MusicBee. Going forward, In Linux, I would like to retain consistency without having to rescan the entire collection.

Would Deadbeef developers consider providing more "under-the-hood" controls for the ReplayGain scanner, such as a slider to adjust target level?

**Comment by Oleksiy-Yakovenko** [comment]
Correct, it's -23.
Yes, it can be considered.

**Reply by garuffalo** [reply]
Thank you for the reply. To me, that would be a useful feature. There doesn't seem to be much on Linux that has ReplayGain processing capability.

Even if DeaDBeeF was hard coded to use -18 LUFS, it would be consistent with the implementation on such venerable apps as Foobar2000, MusicBee, and dBpoweramp. There must be a good reason that they chose -18 LUFS as a reference value.
